src/functionalTest/java/uk/gov/hmcts/reform/pip/account/management/controllers/AccountTest.java

```diff
@@ -118,8 +118,8 @@ class AccountTest {
     private static final String SURNAME = "Surname";
     private static final UserProvenances PROVENANCE = UserProvenances.PI_AAD;
     private static final Roles ROLE = Roles.INTERNAL_ADMIN_CTSC;
-    private static final String ISSUER_EMAIL = "issuer@email.com";
-    private static final String ISSUER_HEADER = "x-issuer-email";
+    private static final String ISSUER_ID = "issuer@email.com";
```

Comment:
Should we change the value to an ID for ISSUER_ID?

Reply:
Good spot, updated :)